.github/workflows/deploy-lambdas.yml

```diff
@@ -0,0 +1,102 @@
+name: Deploy lambdas
+
+on:
+  workflow_call:
+    inputs:
+      ENV:
+        required: true
+        type: string
+      SERVICE:
+        required: true
+        type: string
+    secrets:
+      SLACK_NOTIFICATION_CI_RUNS:
+        required: true
+      ACCESS_TOKEN:
+        required: true
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      AWS_DEFAULT_REGION:
+        required: true
+
+jobs:
+  deploy:
+    name: Deploy Staging Lambdas
+    runs-on: ubuntu-latest
+    container:
+      image: timbru31/node-alpine-git
```

Comment:
I assume that you chose `timbru31/node-alpine-git` because it based on the official node alpine image with the benefit of having git installed. Do you think that the comfort eliminating the `apk install git` is worth it compared to using the official node image which offers more version and is more up to date (as a consequence more secure under some circumstances).

Reply:
It's probably better to use the official one and use apk to add git, I was just being lazy